INPTPZ1/Polynomial.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections.Generic;
+
+
+namespace INPTPZ1
+{
+    class Polynomial
+    {
+        public List<ComplexNumber> Coefficient { get; set; }
+
+        public Polynomial()
+        {
+            Coefficient = new List<ComplexNumber>();
+        }
+
+        public Polynomial Derive()
+        {
+            Polynomial polinomial = new Polynomial();
+            for (int i = 1; i < Coefficient.Count; i++)
+            {
+                var tempComplexNumber = new ComplexNumber()
+                {
+                    Real = i
+                };
+                var coefficient = Coefficient[i].Multiply(tempComplexNumber);
+                polinomial.AddСoefficient(coefficient);
+            }
+
+            return polinomial;
+        }
+
+        public ComplexNumber Eval(ComplexNumber operand2)
```

Comment:
Eval is method and not binary operator. Thus name "operand2" is really misleading.